frontend/src/components/modals/settings/SettingsForm.tsx

```diff
@@ -54,8 +54,6 @@ function SettingsForm({
 
   const handleAdvancedChange = (value: boolean) => {
     setEnableAdvanced(value);
-    // Reset the base URL if the user disables advanced options
-    if (!value) onBaseURLChange("");
```

Comment:
Reverting this for now--I think we need to do this for all advanced settings or none of them